playbooks/infra/deploy-bm-hypervisor.yml

```diff
@@ -0,0 +1,225 @@
+---
+- name: Create a Kickstart-enabled ISO
+  hosts: bastion
+  gather_facts: true
+  vars:
+    iso_mount_path: "{{ dest_iso_dir }}/mount"
+    os_install_path: "{{ dest_iso_dir }}/os-install"
+    location: rdu
+    system_iso_link: "{{ system_iso_rdu_link if location == 'rdu' else system_iso_tlv_link }}"
+  tasks:
+    - name: Set ISO name
+      ansible.builtin.set_fact:
+        iso_name: installation.iso
+
+    - name: Set hypervisor net config variable
+      ansible.builtin.set_fact:
+        hypervisor_net_config: "{{ hostvars['hypervisor'].net_config | from_yaml }}"
+
+    - name: Prepare kickstart iso
+      ansible.builtin.import_role:
+        name: kickstart_iso
+      vars:
+        kickstart_iso_link: "{{ system_iso_link }}"
+        kickstart_iso_name: "{{ iso_name }}"
+        kickstart_iso_file_desire_location: /opt/http_store/data
+        kickstart_iso_timezone: "{{ hostvars['hypervisor'].timezone }}"
+        kickstart_iso_password: "{{ ansible_password }}"
+        kickstart_iso_username: "{{ ansible_user }}"
+        kickstart_iso_net_config: "{{ hypervisor_net_config }}"
+
+    - name: Set UP http storage
+      ansible.builtin.import_role:
+        name: redhatci.ocp.setup_http_store
```

Comment:
Maybe we should also use `validate_http_store` here

Reply:
This role check http share between **http_store** and **bastion** hosts. In my case I have only 1 active machine which is **bastion** host so I can not use this role. I managed to redeploy BM dozens of times without validate_http_store so I think we don't really need it here.